codecov.yml

```diff
@@ -2,24 +2,25 @@ codecov:
   require_ci_to_pass: yes
 coverage:
   status:
-    patch: no
+    patch: off
+    changes: off
     project:
       default:
         threshold: 1%
         if_not_found: success
-    changes: no
+  round: down
   precision: 2
-  range: "50...100"
+  range: "50..75"
 ignore:
- - "test/.*"
- - ".github/.*"
- - ".mvn/.*"
- - ".style/.*"
- - "*.md"
+ - "**/test"
```

Comment:
more upper folder?

Does this include the following cases?

server/test/xxx

Reply:
yes, this rule will ignore any folders and files that call `test`. The example: https://docs.codecov.com/docs/ignoring-paths#ignoring-specific-file-names